src/Dialogs/Device/DeviceEditWidget.cpp

```diff
@@ -78,6 +78,14 @@ FillPress(DataFieldEnum &dfe) noexcept
   dfe.addEnumText(_T("Pitot (airspeed)"), (unsigned)DeviceConfig::PressureUse::PITOT);
 }
 
+static void
+FillRPMScaling(DataFieldEnum &dfe) noexcept
+{
+  dfe.addEnumText(_T("2S1I"), 10);
+  dfe.addEnumText(_T("2S2I"), 5);
+  dfe.addEnumText(_T("4S1I"), 20);
```

Comment:
A 4-stroke engine has 20 revs per ignition?

Reply:
No, because there is no float, I use those values divided by 10.0 to get scaling from 0.5 to 2.0.
I know it is a workaround, but i could not come up with something better...sorry